exercises/prime-factors/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub fn factors(n: u32) -> Vec<u32> {
+    unimplemented!()
```

Comment:
Please update to reference `n` so that there is no unused variable warning, [like this](https://github.com/exercism/rust/pull/438#issuecomment-368357841).

Reply:
Oh nice! I didn't know you could do that. :+1: